pgconf-nyc24.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Meet Xata at PGConf NYC 2024'
+description: 'Join Xata at PGConf NYC 2024 as we showcase our serverless Postgres platform and present key talks on schema migrations and PostgreSQL operational pain points and solutions.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pgconf-nyc@2x.jpg
+  alt: PGConf NYC
+author: Cezzaine Zaher
+date: 09-17-2024
+tags: ['community', 'events', 'postgres']
+published: true
+slug: pgconf-nyc2024
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pgconf-nyc-og@2x.jpg
+---
+
+Xata is heading to New York this month as a Gold Sponsor of the upcoming [PGConf NYC 2024](https://2024.pgconf.nyc), a three day event for the PostgreSQL community. As a company that builds on the strengths of Postgres, this conference is particularly meaningful to us—and we’re looking forward to contributing to the vibrant discussions and innovation that surround the PostgreSQL community.
+
+PGConf NYC brings together developers, database administrators, engineers, and open-source enthusiasts from across the globe, offering a unique opportunity to learn from experts, share experiences, and deep dive into the technical aspects of PostgreSQL. With Postgres continuing to grow as one of the most popular databases in the world, these community-focused events are important in shaping future innovations.
+
+At Xata, we’re passionate about Postgres and committed to making it even more accessible and powerful for developers. Xata’s value proposition is simple: we bring the best of PostgreSQL and combine it with the [power of a serverless platform](https://xata.io/serverless-postgres). This means no more worrying about scaling, infrastructure, or managing your database—Xata takes care of that, letting developers focus on what they do best: building.
+
+At PGConf NYC, we’ll be showcasing how Xata makes working with Postgres not only easier but also more productive. Whether you’re interested in learning more about how to handle large datasets, streamline your workflows, or take advantage of the serverless revolution, our team will be on hand to answer your questions.
+
+# Xata’s featured talks
+
+We’re hosting two technical talks at the conference that dive into some of the key challenges and solutions in the Postgres world.
+
+## Postgres schema migrations using the expand/contract pattern
+
+_Monday, 30th September – 15:00 (The Hub)_
```

Comment:
I wonder if these would be better as call outs like in the other community blog we did?

![CleanShot 2024-09-17 at 09 40 41@2x](https://github.com/user-attachments/assets/f1b88fa4-be37-4741-aee8-0063e1d7de49)

Here's a link to the code snippet if you like it: https://github.com/xataio/mdx-blog/blob/c64a96880535634910e0f13606a427f4d82fa14b/oss-vienna-happy-hour.mdx?plain=1#L92